src/library/scala/collection/LinearSeq.scala

```diff
@@ -184,7 +220,6 @@ trait LinearSeqOps[+A, +CC[X] <: LinearSeq[X], +C <: LinearSeq[A] with LinearSeq
     last
   }
 
-  /** $willForceEvaluation */
```

Comment:
Has this comment been moved to the base method definition? I don’t find it.

Reply:
It turns out that it doesn't force evaluation, so the comment was wrong. I will have a look at the base definition to see if it forces evaluation, but I suspect it doesn't

Reply:
It seems like the base definition only forces evaluation if the `tail` method forces evaluation (the same is true for `inits`/`init`). Since `tail` is not marked as `$mayForceEvaluation`, I am personally disinclined to mark `tails` as such.